testing_and_setup/compass/ocean/cosine_bell/QU120/default/build_base_mesh.py

```diff
@@ -19,3 +20,12 @@ def cellWidthVsLatLon():
 
     cellWidth = constantCellWidth * np.ones((lat.size, lon.size))
     return cellWidth, lon, lat
+
+
+def main():
+    cellWidth, lon, lat = cellWidthVsLatLon()
+    build_spherical_mesh(cellWidth, lon, lat, out_filename='base_mesh.nc')
```

Comment:
I reorganized the process for creating spherical meshes in COMPASS a little bit.  It was really problematic the way we were doing it before because we were trying to call a local version of a function within a module (`define_base_mesh.cellWidthVsLatLon()`) but that's not really an accepted approach for writing python.  One major problem is that a copy of the module behaves differently from a link.

A much cleaner approach is to have a local script that is the top level.  It first calls whatever function it likes to define the `cellWidth`, `lon` and `lat`, and then passes this information on to a COMPASS function `build_spherical_mesh`.  No local module, just a script and no issues with linking to the script rather than copying it.